contrib/platform/src/com/sun/jna/platform/unix/aix/Perfstat.java

```diff
@@ -0,0 +1,1040 @@
+/* Copyright (c) 2020 Daniel Widdis, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.unix.aix;
```

Comment:
We currently have linux on the toplevel and solaris as a package under unix. I'm fine with keeping the package as is, just curious.

Reply:
In my own project I have a `unix` package with common unix headers/functions (usually POSIX) and sub-packages for `freebsd`, `solaris`, and now `aix`.  I haven't contributed any of my `freebsd` mappings (minor type bitwidth differences in timeval structures) to JNA but if I did they'd probably live in a similar `unix` subpackage.   In theory, `mac` or `darwin` should also reside under `unix` but no reason to move it from where it is.